earn/src/pages/AdvancedPage.tsx

```diff
@@ -395,8 +388,8 @@ export default function AdvancedPage() {
           <NewSmartWalletModal
             availablePools={availablePools}
             defaultPool={defaultPool}
-            isOpen={newSmartWalletModalOpen}
-            setIsOpen={setNewSmartWalletModalOpen}
+            isOpen={openedModal === OpenedModal.NewSmartWallet}
+            setIsOpen={(_open) => setOpenedModal(_open ? OpenedModal.NewSmartWallet : OpenedModal.None)}
```

Comment:
Could we use a different argument name, in typescript, an underscore prefix for arguments indicates they are intentionally unused which is not the case here.